analyzers/tests/SonarAnalyzer.UnitTest/Rules/EqualityOnFloatingPointTest.cs

```diff
@@ -25,8 +25,22 @@ namespace SonarAnalyzer.UnitTest.Rules
     [TestClass]
     public class EqualityOnFloatingPointTest
     {
+        private readonly VerifierBuilder builder = new VerifierBuilder<EqualityOnFloatingPoint>();
+
         [TestMethod]
         public void EqualityOnFloatingPoint() =>
-            new VerifierBuilder<EqualityOnFloatingPoint>().AddPaths("EqualityOnFloatingPoint.cs").Verify();
+            builder.AddPaths("EqualityOnFloatingPoint.cs").Verify();
+
+#if NET7_0_OR_GREATER
+
+        [TestMethod]
+        public void EqualityOnFloatingPoint_CSharp11() =>
```

Comment:
This test should not be specific to C# 11  - but it does not work for .NET Framework.
Maybe rename it to `EqualityOnFloatingPoint_NetCore`? Not sure what is convention.

Reply:
It uses `IFloatingPointIeee754`. That interface uses C#11 features like `public static TResult operator == (TSelf? left, TOther? right);`. So it is C# 11.